.golangci.yml

```diff
@@ -166,3 +167,43 @@ issues:
     # that's a valid usage of bson.D
     - linters: [govet]
       text: "`go.mongodb.org/mongo-driver/bson/primitive.E` composite literal uses unkeyed fields"
+
+    # integration tests exclusions
+    - path: _test\.go
+      text: "composites: go.mongodb.org/mongo-driver/bson/primitive.E struct literal uses unkeyed fields"
+      linters: [govet]
+    - path: _test\.go
+      linters: [lll]
+    - path: _test\.go
+      text: "import of package `go.mongodb.org/mongo-driver/*"
```

Comment:
That's very unfortunate. On the one hand, we don't want driver dependency in the main module: #853. On the other, it is not possible to specify "integration" directory there.

I'm not sure what the answer is there. Use separate configuration files and sync them manually? Do something with YAML includes?.. Meh.

@rumyantseva WDYT?

Reply:
@AlekSi yes, this is a little weird. IMO would be clean enough to create separate configuration file for the integration module.

Reply:
Ok, let's use separate configuration files for the integration module without any templating for now.

@ravilushqa Wanna finish that one for Hacktoberfest? :)

Reply:
Ah, I missed that Hacktoberfest started 😄 
Sure, it would be great.
However looks like I have to create a new PR, isn't it?

> Your PR/MRs must be created between October 1 and October 31 (in any time zone, UTC-12 thru UTC+14).
Your PR/MRs must be made to a public, unarchived repository.

Reply:
I think it will be easier, yeah